web/src/user/UserInterface.ts

```diff
@@ -192,85 +193,18 @@ export class UserInterface extends Interface {
-                                  >
-                                      <button
-                                          class="pf-c-button pf-m-plain"
-                                          type="button"
-                                          aria-label="${msg("Unread notifications")}"
-                                          @click=${() => {
-                                              this.notificationDrawerOpen =
-                                                  !this.notificationDrawerOpen;
-                                              updateURLParams({
-                                                  notificationDrawerOpen:
-                                                      this.notificationDrawerOpen,
-                                              });
-                                          }}
-                                      >
-                                          <span
-                                              class="pf-c-notification-badge ${this
-                                                  .notificationsCount > 0
-                                                  ? "pf-m-unread"
-                                                  : ""}"
-                                          >
-                                              <pf-tooltip
-                                                  position="top"
-                                                  content=${msg("Open Notification drawer")}
-                                              >
-                                                  <i class="fas fa-bell" aria-hidden="true"></i>
-                                              </pf-tooltip>
-                                              <span class="pf-c-notification-badge__count"
-                                                  >${this.notificationsCount}</span
-                                              >
-                                          </span>
-                                      </button>
-                                  </div> `
-                                : html``}
-                            ${this.uiConfig.enabledFeatures.settings
-                                ? html` <div class="pf-c-page__header-tools-item">
-                                      <a
-                                          class="pf-c-button pf-m-plain"
-                                          type="button"
-                                          href="#/settings"
-                                      >
-                                          <pf-tooltip position="top" content=${msg("Settings")}>
-                                              <i class="fas fa-cog" aria-hidden="true"></i>
-                                          </pf-tooltip>
-                                      </a>
-                                  </div>`
-                                : html``}
+                            ${this.renderApiDrawer()}
+                            <!-- -->
```

Comment:
These are here just to space things out; otherwise, `prettier` wants to string all these together into a single line, which was harder to read.